src/redis-cli.c

```diff
@@ -1809,6 +1818,8 @@ static void usage(void) {
 "                     (if both are used, this argument takes precedence).\n"
 "  --user <username>  Used to send ACL style 'AUTH username pass'. Needs -a.\n"
 "  --pass <password>  Alias of -a for consistency with the new --user option.\n"
+"  --from-user <username>  Source node auth user name where we import keys from.\n"
+"  --from-pass <password>  Source node auth password where we import keys from.\n"
```

Comment:
I don't think this belongs here - it should be generated by `clusterManagerCommandHelp()`, right?

Reply:
@itamarhaber Thanks Itamar, I also just noticed this one did not changed lol.  fixed..

Reply:
Thanks.